blueoil/templates/lmnet/classification.tpl.py

```diff
@@ -80,7 +80,13 @@
 POST_PROCESSOR = None
 
 NETWORK = EasyDict()
-NETWORK.OPTIMIZER_CLASS = tf.train.MomentumOptimizer
+
+if '{{optimizer}}' == 'GradientDescentOptimizer':
+    NETWORK.OPTIMIZER_CLASS = tf.train.GradientDescentOptimizer
+elif '{{optimizer}}' == 'MomentumOptimizer':
+    NETWORK.OPTIMIZER_CLASS = tf.train.MomentumOptimizer
+elif '{{optimizer}}' == 'AdamOptimizer':
+    NETWORK.OPTIMIZER_CLASS = tf.train.AdamOptimizer
 
 if '{{learning_rate_setting}}' != 'fixed':
     NETWORK.OPTIMIZER_KWARGS = {"momentum": 0.9}
```

Comment:
Maybe `AdamOptimizer` and `GradientDescentOptimizer` don't accept `{"momentum": 0.9}`, but I'm not sure.

https://www.tensorflow.org/api_docs/python/tf/train/GradientDescentOptimizer
https://www.tensorflow.org/api_docs/python/tf/train/AdamOptimizer

I will verify it's work well when optimizer is `GradientDescentOptimizer` or `AdamOptimizer`